lib/node_modules/@stdlib/blas/ext/base/sindex-of/benchmark/benchmark.length.js

```diff
@@ -0,0 +1,96 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var isInteger = require( '@stdlib/assert/is-integer' ).isPrimitive;
+var oneTo = require( '@stdlib/array/one-to' );
+var pkg = require( './../package.json' ).name;
+var sindexOf = require( './../lib' );
+
+
+// FUNCTIONS //
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - array length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len ) {
+	var x = oneTo( len, 'float32' );
+	return benchmark;
+
+	/**
+	* Benchmark function.
+	*
+	* @private
+	* @param {Benchmark} b - benchmark instance
+	*/
+	function benchmark( b ) {
+		var out;
+		var i;
+
+		b.tic();
+		for ( i = 0; i < b.iterations; i++ ) {
+			out = sindexOf( x.length, 2.0, x, 1 );
```

Comment:
```suggestion
			out = sindexOf( x.length, len+1, x, 1 );
```

`2.0` obviously does not make sense here, as then the benchmarks are effectively constant time. You should be benchmarking the worst case scenario (i.e., when you need to search an entire array).